README.md

```diff
@@ -10,10 +10,12 @@ npm install --save @vuedoc/parser
 ```
 
 ## Options
-- `filename` (*required*) The filename to parse
+- `filename` (*required* unless `filecontent` is passed) The filename to parse
+- `filecontent` (*required* unless `filename` is passed) The string to parse
 - `encoding` (*optional*) `default: utf8`
 - `ignoreName` (*optional*) `default: false` Set `true` to ignore the component name in the result
 - `ignoreDescription` (*optional*) `default: false` Set `true` to ignore the component description in the result
+- `defaultMethodVisibility` (*optional*) `default: DEFAULT_VISIBILITY`. Can be set to `private`, `public`, or `protected` to override global `DEFAULT_VISIBILITY` for component methods.
```

Comment:
`DEFAULT VISIBILITY` is an internal variable, the user does not know it and don't know its value.
I suggest to set `default: 'public'`